modules/plugin/jdbc/jdbc-postgis/src/main/java/org/geotools/data/postgis/FilterToSqlHelper.java

```diff
@@ -703,23 +704,66 @@ public String buildJsonFromStrPointer(String[] pointers, int index, Expression e
         }
     }
 
+    /**
+     * Encode the jsonArrayContains call using the jsonb_path_exists function from Postgres
+     *
+     * @param jsonArrayContains
+     * @throws IOException
+     */
     private void encodeJsonArrayContains(Function jsonArrayContains) throws IOException {
+        String useJsonPathExists =
+                System.getProperty("org.geotools.data.postgis.useJsonPathExists");
```

Comment:
If you keep the variable, please use a constant for "org.geotools.data.postgis.useJsonPathExists" and use it consistently here and in the test code.

Reply:
Removed it